package.json

```diff
@@ -5,15 +5,57 @@
         "frontends/*"
     ],
     "scripts": {
-        "start:dev": "yarn workspaces foreach --parallel --interlaced run start:dev",
-        "build": "yarn workspaces foreach run build",
-        "lint": "yarn workspaces foreach run lint",
-        "fmt": "yarn workspaces foreach run fmt",
-        "fmt:check": "yarn workspaces foreach run fmt:check",
-        "scss_lint": "yarn workspaces foreach run scss_lint",
-        "test": "yarn workspaces foreach run test",
-        "typecheck": "yarn workspaces foreach run typecheck"
+        "foreach-with-fallback": "yarn workspaces foreach --verbose --topological exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js $0 $1",
+        "postinstall": "yarn run build",
+        "start-dev": "yarn workspaces foreach --verbose --parallel --interlaced run start-dev",
+        "build": "run foreach-with-fallback build global:build",
+        "clean": "run foreach-with-fallback clean global:clean",
+        "lint-check": "run foreach-with-fallback lint-check global:lint-check",
+        "lint-fix": "run foreach-with-fallback lint-fix global:lint-fix",
+        "fmt-check": "run foreach-with-fallback fmt-check global:fmt-check",
+        "fmt-fix": "run foreach-with-fallback fmt-fix global:fmt-fix",
+        "test": "jest && yarn workspace open-discussions run test",
+        "scss-lint": "run foreach-with-fallback scss-lint global:scss-lint",
+        "typecheck": "run foreach-with-fallback typecheck global:typecheck",
+        "global:build": "cd $INIT_CWD && yarn global:clean && yarn global:compile && yarn global:build-flowtypes",
+        "global:start-dev": "cd $INIT_CWD && yarn global:clean && yarn global:compile-watch",
+        "global:clean": "cd $INIT_CWD && rimraf -rf build/ tsconfig.tsbuildinfo",
+        "global:compile": "cd $INIT_CWD && tsc",
+        "global:compile-watch": "cd $INIT_CWD && tsc --watch",
+        "global:coverage": "cd $INIT_CWD && yarn test --coverage",
+        "global:fmt-check": "cd $INIT_CWD && prettier --no-semi --check --with-node-modules src/**/*.{ts,tsx}",
+        "global:fmt-fix": "cd $INIT_CWD && prettier --no-semi --with-node-modules --write src/**/*.{ts,tsx}",
```

Comment:
`--with-node-modules` is important... Prettier seems unable to find the src files without it. I suspect this is related to the way that `yarn` puts symlinks to local packages into node modules.